airsonic-main/src/main/java/org/airsonic/player/controller/StreamController.java

```diff
@@ -231,25 +233,37 @@ public void handleRequest(HttpServletRequest request, HttpServletResponse respon
             try (OutputStream out = makeOutputStream(request, response, range, isSingleFile, player, settingsService)) {
                 final int BUFFER_SIZE = 2048;
                 byte[] buf = new byte[BUFFER_SIZE];
+                long bytesWritten = 0;
 
                 while (!status.terminated()) {
                     if (player.getPlayQueue().getStatus() == PlayQueue.Status.STOPPED) {
                         if (isPodcast || isSingleFile) {
                             break;
                         } else {
-                            sendDummy(buf, out);
+                            sendDummyDelayed(buf, out);
```

Comment:
Out of curiousity, any idea what the goal of sleeping here is? It's probably inherited from Subsonic and I'm probably missing something, but `Thread.sleep` feels like a hacky workaround...

Reply:
I think originally that code was meant to keep the stream/connection alive while waiting for something else to occur (but I'm not really sure what that was).

Reply:
Well, in any case I don't see why we shouldn't keep it here for now until we figure out what this means.